tests/suit/tests_suite_test.go

```diff
@@ -0,0 +1,101 @@
+package suit_test
+
+import (
+	"flag"
+	"fmt"
+	"github.com/konveyor/forklift-controller/tests/suit/framework"
+	"github.com/konveyor/forklift-controller/tests/suit/reporters"
+	"github.com/konveyor/forklift-controller/tests/suit/utils"
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"testing"
+	"time"
+)
+
+const (
+	pollInterval     = 2 * time.Second
+	nsDeletedTimeout = 270 * time.Second
+)
+
+var (
+	kubectlPath       = flag.String("kubectl-path", "/home/eslutsky/bin/kubectl", "The path to the kubectl binary")
```

Comment:
`eslutsky` path is temporary, right?

Reply:
yes, it's not a good default but that's part of the cleanup I referred to

Reply:
yes, but we are not using kubectl cmd , i'll remove this part